lr_face/models.py

```diff
@@ -54,5 +55,96 @@ def score_for_pair(self, pair):
         img2 = resize_and_normalize(pair[1], self.model.input_shape[1:3])
         img1_representation = self.model.predict(img1)[0, :]
         img2_representation = self.model.predict(img2)[0, :]
-        score = spatial.distance.cosine(img1_representation, img2_representation)
+        score = spatial.distance.cosine(img1_representation,
+                                        img2_representation)
         return score
+
+
+class FinetuneModel(tf.keras.Model):
+    """
+    A subclass of tf.keras.Model that can be used to finetune the pre-trained
+    embedding models.This new training model takes 3 inputs, namely:
```

Comment:
Enigszins ambigu omdat zowel init als call argumenten nemen. Misschien toevoegen iets als. 
This class wraps a particular embedding model 'finetuner = TripletEmbedder(embedding_model)'. When called, the Embedder takes 3 inputs, namely: ...

Reply:
Ik kan het commentaar over de inputs inderdaad beter hier weghalen, deze docstring op class-niveau moet meer een high-level uitleg zijn over wat deze class doet zonder al te veel op de implementatie van de individuele methodes in te gaan.